docs/make.jl

```diff
@@ -12,4 +12,6 @@ makedocs(
 deploydocs(
     repo = "github.com/JuliaAstro/FITSIO.jl.git",
     target = "build",
+    deps   = nothing,
```

Comment:
Are you sure we don't need `deps   = Deps.pip("mkdocs", "python-markdown-math")` 
 here, following https://juliadocs.github.io/Documenter.jl/stable/man/hosting/#The-deploydocs-Function-1 ? I would imagine that otherwise mkdocs is not actually available.

Reply:
Corrected.